### ISSUE TITLE
ColorControl 9.6.0.0 always detects a new update

### ISSUE THREAD

**Issue by javsp** [Maassoft/ColorControl#238]
I've just updaated to 9.6.0.0 but ColorControl keeps warning there is a new update (the same 9.6.0.0).

It seems it's ignoring the last update it's already installed.

**Comment by Maassoft** [comment]
Yes, you're right, there's something wrong in the version comparer. Will be fixed in next release.
For now I suggest temporarily disabling the automatic check for updates.

**Comment by Maassoft** [comment]
This should be fixed now with 9.6.1.0.

**Comment by javsp** [comment]
Yeah, it works now. Thanks!
